File: app/src/main/java/com/rr/rhythms/Business/Graphs/Graph24.java
package com.rr.rhythms.Business.Graphs;

import android.graphics.Color;

/**
 * Created by Huruk on 7/5/2016.
 */
public class Graph24 extends GraphBase {

    public static final int PERIOD = 24;

    @Override
    public int GetPeriod() {
        return PERIOD;
    }

    @Override
    public int getColor() {
        return Color.RED;
    }

    @Override
    public String getName() {
        return Integer.toString(PERIOD);
    }
}

File: app/src/main/java/com/rr/rhythms/Business/Graphs/Graph24.kt
package com.rr.rhythms.Business.Graphs

import android.graphics.Color

/**
 * Created by Huruk on 7/5/2016.
 */
class Graph24 : GraphBase() {
    override val color: Int = Color.RED

    override val name: String = period.toString()

    override val period: Int
        get() = 24
}

